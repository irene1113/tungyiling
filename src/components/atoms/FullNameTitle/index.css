.full-name-title {
	display: inline-flex;
	flex-direction: column;
	align-items: flex-end;

	& > span {
		display: inline-flex;
		line-height: 0.75;
		font-family: 'Roboto', sans-serif;
		font-weight: bold;
		color: var(--black);

		&:not(:last-child) {
			margin-bottom: 12px;
		}
	}
}
