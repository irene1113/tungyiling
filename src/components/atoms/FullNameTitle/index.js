import React from 'react';
import classnames from 'classnames';

import styles from './index.css';

const FullNameTitle = ({ className, fontSize = 110 }) => (
	<div className={classnames(styles.fullNameTitle, className)}>
		<span style={{ fontSize: `${fontSize}px` }}>YILING</span>
		<span style={{ fontSize: `${fontSize}px` }}>TUNG</span>
	</div>
);

export default FullNameTitle;
