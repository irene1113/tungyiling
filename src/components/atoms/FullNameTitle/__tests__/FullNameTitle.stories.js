import React from 'react';
import { storiesOf } from '@storybook/react';
import { number } from '@storybook/addon-knobs';

import FullNameTitle from 'components/atoms/FullNameTitle';

const stories = storiesOf('atoms/FullNameTitle', module);

stories.add('__interactive', () => <FullNameTitle fontSize={number('fontSize', 110)} />);
