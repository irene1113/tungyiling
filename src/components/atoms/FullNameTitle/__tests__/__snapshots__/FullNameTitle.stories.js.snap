// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`__interactive 1`] = `
<DocumentFragment>
  <div
    class="fullNameTitle"
  >
    <span
      style="font-size: 110px;"
    >
      YILING
    </span>
    <span
      style="font-size: 110px;"
    >
      TUNG
    </span>
  </div>
</DocumentFragment>
`;
